src/app/admin/clients/[id]/company-dependants/company-dependants-content.tsx

```
@@ -101,8 +101,7 @@ export default function CompanyDependantsContent({ client }: { client: Client })
                 department: beneficiary.department,
                 role: beneficiary.role,
             }));
-            dependantsWithContext.forEach(d => acc.push(d));
-            return acc;
+            return [...acc, ...dependantsWithContext];
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)